<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Foods</class>
 <widget class="QMainWindow" name="Foods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Foods</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj skladniki, wpisuj je po spacji</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Oto wyszukane dania z podanych składników</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="items">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>140</y>
      <width>769</width>
      <height>459</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-40</y>
       <width>751</width>
       <height>261</height>
      </rect>
     </property>
     <widget class="QWidget" name="img2" native="true">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>421</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="prev">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Poprzedni</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>100</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Nastepny</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="results" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>801</width>
       <height>241</height>
      </rect>
     </property>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="autor">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="data_dodania">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>170</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="link">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>201</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Zobacz pelny opis</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Szukaj</string>
    </property>
   </widget>
   <widget class="QLabel" name="item_count">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
